app/src/main/java/com/duckduckgo/app/privacymonitor/ui/NetworksRenderer.kt

```diff
@@ -40,11 +40,27 @@ class NetworksRenderer {
 
     @DrawableRes
     fun networkPillIcon(context: Context, networkName: String): Int? {
-        val drawable = "network_pill_$networkName"
+        return networkIcon(context, networkName, "network_pill_")
+    }
+
+    @DrawableRes
+    fun networkLogoIcon(context: Context, networkName: String): Int? {
+        return networkIcon(context, networkName, "network_logo_")
```

Comment:
Can we add a new entry in `lint.xml` otherwise these dynamic ones get flagged as unused. There's already a rule for the pill icons in there as an example.